docs/bis/domains/Provenance-in-BIS.md

```diff
@@ -0,0 +1,75 @@
+# Provenance in BIS
+
+## Introduction
+
+Provenance concerns the ability to trace an element in a BIS repository back to its native, external source. Several classes are available in the BisCore schema to help capturing provenance information for Elements in a BIS repository. They are particularly important for data-synchronization workflows, while detecting changes in an external source with respect to data stored in a BIS repository.
+
+## Core Concepts
+
+The following class diagrams depict the main classes from the BisCore schema involved in capturing provenance in BIS repositories. See [Class-diagram Conventions](../guide/references/class-diagram-conventions.md) for details about the conventions used.
+
+![Core Classes](../guide/media/external-sources-classes.png)
+
+![SynchronizationConfigLink](../guide/media/external-sources-classes-1.png)
+
+### RepositoryLink
+
+A [RepositoryLink](./BisCore.ecschema.md#repositorylink-repository-link-badge-textentityclass-kindinfo) represents a link to an external repository. External Repositories may be associated with Reality data as well as with data that is synchronized with elements in a BIS repository.
+
+### ExternalSource
+
+An [ExternalSource](./BisCore.ecschema.md#externalsource-source-badge-textentityclass-kindinfo) represents an `Information Container` within a repository. External repositories are referenced by [RepositoryLink](#repositorylink) instances in a BIS repository.
+
+Some external repositories do not have any data-container concept and therefore, their corresponding [RepositoryLink](#repositorylink) instances will only be associated with one  `ExternalSource` instance.
+
+### ExternalSourceAttachment
+
+An [ExternalSourceAttachment](./BisCore.ecschema.md#externalsourceattachment-external-source-attachment-badge-textentityclass-kindinfo) enables the representation of hierarchical `Information Container`s or acyclical dependency graphs that can be encountered in some external repositories.
+
+`ExternalSourceAttachment` instances are owned by a `ExternalSource` at the higher level of the hierarchy via the [ExternalSourceOwnsAttachments](./BisCore.ecschema.md#externalsourceownsattachments-badge-textrelationshipclass-kindinfo) relationship.
+
+An instance of `ExternalSourceAttachment` also captures any transformations needed - in the form of translation, scale or rotation - relative to the  `ExternalSource` attaching it.
+
+### ExternalSourceAspect
+
+An [ExternalSourceAspect](./BisCore.ecschema.md#externalsourceaspect-external-source-aspect-badge-textentityclass-kindinfo) is an [Element Aspect](../guide/fundamentals/elementaspect-fundamentals.md) which is meant to capture the specific details with regards to provenance for its owning Element.
+
+Provenance for a specific Element is captured in terms of:
+
+- An `Identifier` in the source repository.
+- A `Kind` of object within the source repository.
```

Comment:
Should we specify the standard kinds that we use and describe their purpose?

Reply:
There are no standard values for `Kind` in BIS. It is rather an implementation detail per Connector. I would expect that detail to be addressed by each specific Connector documentation, listing what values for `Kind` are stored and their meaning.